python_modules/dagster/dagster/__init__.py

```diff
@@ -520,7 +524,10 @@
     deserialize_value as deserialize_value,
     serialize_value as serialize_value,
 )
-from dagster._utils import file_relative_path as file_relative_path
+from dagster._utils import (
+    file_relative_path as file_relative_path,
+    script_relative_path as script_relative_path,
```

Comment:
ild rather not export this, it does some cute `inspect.stack` shenanigans and `file_relative_path(__file__` isn't that much worse

Reply:
good call